lib/commands/preload.ts

```diff
@@ -201,14 +202,7 @@ Can be repeated to add multiple certificates.\
 			);
 		}
 
-		let certificates: string[];
-		if (Array.isArray(options['add-certificate'])) {
-			certificates = options['add-certificate'];
-		} else if (options['add-certificate'] === undefined) {
-			certificates = [];
-		} else {
-			certificates = [options['add-certificate']];
-		}
+		const certificates: string[] = options['add-certificate'] || [];
```

Comment:
With oclif `multiple: true`, `options['add-certificate']` will be either `undefined` or an array (never a string).